python/test/unit/language/test_core.py

```diff
@@ -736,6 +736,71 @@ def kernel(X, stride_xm, stride_xk,
         assert 'mma.sync.aligned.m16n8k8.row.col.f32.tf32.tf32.f32' not in ptx
 
 
+@pytest.mark.parametrize("BLOCK_M, BLOCK_N, BLOCK_K, num_stages, AT, BT",
+                         [(BLOCK_M, BLOCK_N, BLOCK_K, num_stages, AT, BT)
+                          for BLOCK_M in [64, 128]
+                          for BLOCK_N in [64, 128]
+                          for BLOCK_K in [64, 128]
+                          for num_stages in [1, 2, 3, 4]
+                          for AT in [True, False]
+                          for BT in [True, False]])
+def test_int8_dot(BLOCK_M, BLOCK_N, BLOCK_K, num_stages, AT, BT):
```

Comment:
Hmm do we really need a separate function here? couldn't we re-use the `test_dot` ?

Reply:
Yes. I think it's better to re-use `test_dot`!